packages/client/default.conf

```diff
@@ -0,0 +1,48 @@
+server {
+    listen       80;
+    listen  [::]:80;
+    server_name  localhost;
+
+    #access_log  /var/log/nginx/host.access.log  main;
+
+    location / {
+        root   /usr/share/nginx/html;
+
+        index  index.html index.htm;
+        
+        try_files $uri $uri/ /index.html;
+    }
+
+    #error_page  404              /404.html;
+
+    # redirect server error pages to the static page /50x.html
+    #
+    error_page   500 502 503 504  /50x.html;
+
+    location = /50x.html {
+        root   /usr/share/nginx/html;
+    }
+
+    # proxy the PHP scripts to Apache listening on 127.0.0.1:80
```

Comment:
We don't use PHP. Is this needed?

Reply:
I've just copied the default nginx configuration from the Docker container and added the redirect back to the index.html on line 13. We ourselves are not using PHP, that is correct